qubes-rpc/post-install.d/10-qubes-core-agent-features.sh

```diff
@@ -16,19 +15,18 @@ if [ -r /etc/os-release ]; then
     fi
     qvm-features-request os-distribution="$distro"
 
-    version=$(grep ^VERSION_ID= /etc/os-release)
-    version=${version#VERSION_ID=}
-    version=${version#\"}
-    version=${version%\"}
+    distro_like=$(awk -F '=' '/^ID_LIKE=/{ print $2 }' /etc/os-release)
+    qvm-features-request os-distribution-like="${distro_like}"
```

Comment:
This needs also the backend side, see https://github.com/QubesOS/qubes-core-admin/pull/578 and later https://github.com/QubesOS/qubes-core-admin/pull/585.

What is the "extensive list of supported distributions of Debian" what you can find on Qubes?

Reply:
> This needs also the backend side, see [QubesOS/qubes-core-admin#578](https://github.com/QubesOS/qubes-core-admin/pull/578) and later [QubesOS/qubes-core-admin#585](https://github.com/QubesOS/qubes-core-admin/pull/585).

Will take care of this soon.

> What is the "extensive list of supported distributions of Debian" what you can find on Qubes?

That I can find on Qubes by default is only `kali` and `whonix`. That someone can install from a 3rd party template, that is higher.